uniffi_core/src/ffi/foreigncallbacks.rs

```diff
@@ -128,8 +128,8 @@ use std::sync::atomic::{AtomicUsize, Ordering};
 /// * The `method` selector specifies the method that will be called on the object, by looking it up in a list of methods from
 ///   the IDL. The index is 1 indexed. Note that the list of methods is generated by at uniffi from the IDL and used in all
 ///   bindings: so we can rely on the method list being stable within the same run of uniffi.
-/// * `args` is a serialized buffer of arguments to the function. UniFFI will deserialize it before
-///   passing individual arguments to the user's callback.
+/// * `args_data` and `args_len` represents a serialized buffer of arguments to the function.
```

Comment:
I think it might be nice to expand these comments a little for the sake of external bindings maintainers. In the old versuin the language was fairly vague, but at least the arg as a concrete `RustBuffer`. Here it's just a pointer, so the slightly vague comments about it being a "serialized buffer" need a little more context.